CHANGES/9006.feature

```diff
@@ -0,0 +1 @@
+Enable new tasking system on CollectionImport
```

Comment:
This could probably be ` .misc` since I don't think users care about internal compatability with pulpcore